Razer/cheryl/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.razer.cheryl"
+        android:versionCode="1"
+        android:versionName="1.0">
+	<!--
+		TESTS: Ignore ro.vendor.product.name
```

Comment:
Moto G6 has a link to stock ROM explaining why.

In this case, I really don't see why you're doing this, here is the link to cheryl's vendor build.prop
https://github.com/TadiT7/razer_cheryl_dump/blob/cheryl-user-8.1.0-OPM1.171019.011-RZR-180509-5038-release-keys/vendor/build.prop
You should be able to just match on ro.vendor.build.fingerprint